### DISCUSSION TITLE
Add Template to Vercel Template

### DISCUSSION THREAD

**Discussion by samcuxx** [vercel/community#4582] [Help]
### Summary

i have build a portfolio that i want to share on vercel to help developer build a nice portfolio with my starting portofolio template

### Steps to Reproduce

i have build a portfolio that i want to share on vercel to help developer build a nice portfolio with my starting portofolio template

**Comment by amyegan** [comment]
Hey! Thanks for your interest in adding a template to our Template Marketplace. We are not accepting new templates right now. However, you can create your own Deploy Button to make it easy for people to create their own project based on your repo.

https://vercel.com/docs/deployments/deploy-button#

**Comment by jpvajda** [comment]
@amyegan  I'm also trying to add a new template for my company, Deepgram. Is Vercel still not accepting new Templates?  cc @lukeocodes

**Reply by amyegan** [reply]
Thanks for reaching out about this! We're not currently accepting new templates while we reimagine what a new template marketplace could look like. But you can still make your projects available to install on Vercel with the [Deploy Button](https://vercel.com/docs/deployments/deploy-button#) which can be put in your `readme.md` or on your website. It works just like the "Deploy" button on our existing template marketplace pages.

You can generate your own custom Deploy Button using the form at [vercel.com/docs/deployments/deploy-button#generate-your-own](https://vercel.com/docs/deployments/deploy-button#generate-your-own)

**Comment by amyegan** [comment]
Good news! We just re-opened Templates Marketplace for new submissions. If you're still interested in sharing your template, please use the [Templates Marketplace submission form](https://vercel.com/templates/submit) to send us the details. 🙂

**Reply by lukeocodes** [reply]
Thanks @amyegan! We got a few coming your way!
